site/content/docs/main/website-guidelines.md

```diff
@@ -13,21 +13,21 @@ At the root of the project, run:
 make serve-docs
 ```
 
-This runs all the Ruby dependencies in a container.
+This runs all the Hugo dependencies in a container.
 
 Alternatively, for quickly loading the website, under the `velero/site/` directory run:
 
 ```bash
-bundle exec jekyll serve --livereload --future
+hugo serve
```

Comment:
should we change this to `make serve-docs`?

Reply:
Do you mean like the example above? I was going for providing thee make command and hugo command, as it was before with the make command and Jekyll build command.

Reply:
I didn't see that. Thanks for point that out. You can ignore this comment.